commands/wish.js

```diff
@@ -67,72 +76,90 @@ function validateArgs(args, message) {
     c: "character",
     w: "weapon",
     b: "beginner",
-    s: "standard"
-  }
+    s: "standard",
+  };
 
   var banners = Object.values(shorthandBanners);
 
-  if(args.length == 2 || args.length > 4) { return false };
+  if (args.length == 2 || args.length > 4) {
+    return false;
+  }
 
-  if(args[0].length > 1 && !banners.includes(args[0].toLowerCase())) { return false };
-  if(args[0].length == 1 && !banners.map(x => x[0]).includes(args[0].toLowerCase())) { return false };
+  if (args[0].length > 1 && !banners.includes(args[0].toLowerCase())) {
+    return false;
+  }
+  if (
+    args[0].length == 1 &&
+    !banners.map((x) => x[0]).includes(args[0].toLowerCase())
+  ) {
+    return false;
+  }
 
-  if(!/^[0-9]*$/.test(args[1])) { return false };
+  if (!/^[0-9]*$/.test(args[1])) {
+    return false;
+  }
 
-  if(!/((char)|(weap))=\d+$/.test(args[2])) { return false };
-  if(args[3] && !/((char)|(weap))=\d+$/.test(args[3])) { return false };
+  if (!/((char)|(weap))=\d+$/.test(args[2])) {
+    return false;
+  }
+  if (args[3] && !/((char)|(weap))=\d+$/.test(args[3])) {
+    return false;
+  }
 
-  if(args[0].length == 1) { args[0] = shorthandBanners[args[0]] };
+  if (args[0].length == 1) {
+    args[0] = shorthandBanners[args[0]];
+  }
 
   args[0] = args[0].replace(/^\w/, (c) => c.toUpperCase());
 
-  var counts = getCounts(args[2], args[3])
+  var counts = getCounts(args[2], args[3]);
 
   var result = {
     banner: args[0],
     wishes: args[1],
-    chars: counts["chars"] || '0',
-    weapons: counts["weapons"] || '0'
-  }
+    chars: counts["chars"] || "0",
+    weapons: counts["weapons"] || "0",
```

Comment:
Same with the other one I commented on earlier. I don't think there should be a comma here unless there's a value after this.

Reply:
You can disregard this one since we already came up with a resolution in the other line with the same comment.